client_code/Demo/__init__.py

```diff
@@ -17,14 +17,15 @@ def __init__(self, **properties):
         self.init_custom_slider_formatter()
 
         self.progress = 0
-        self.item = self.default_item = dict(
+        self.default_item = dict(
             tally=100,
             counter=0,
             values=self.slider.start,
             agree=self.slider_agree.value,
             chips=["a", "b", "c"],
             text="",
         )
+        self.item = self.default_item.copy()
```

Comment:
Actually, I think the `copy()` is redundant here. Setting `self.item` will pass `self.default_item` to the `dict` function anyway.

Reply:
I know we discussed using `copy()` on #89  but I actually think it's just the docs that need updating.

Reply:
It is redundant with auto_refreshing. But in my original demo app, I was going to show the example with and without auto-refreshing to demonstrate the difference. If others try the same, they may be confused by the lack of reset behavior without auto-refreshing. On the other hand, the exercise will highlight to them the now-documented auto-refreshing side-effect that causes `self.item` to no longer reference the same underlying object as `self.default_item`.

If, hearing my reasoning for wanting to include `copy()`, you'd still rather not, I'll make the change to remove them.

Reply:
I think I'd prefer to leave it out. In your demo app, it might be worth just chucking in a comment or two to explain the difference (perhaps with a link to the docs).